### ISSUE TITLE
Merge `mainnet-liberated` into `Epheus`

### ISSUE THREAD

**Issue by dobertRowneySr** [Joystream/joystream#4556]
SSIA
The mainnet-liberated branch should have https://github.com/Joystream/joystream/pull/4542 
merged

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202078489349362/1203780550680374) by [Unito](https://www.unito.io)

**Comment by dobertRowneySr** [comment]
# Details
- [x] proposal `UpdateChannelPayouts` should be activated
- [x] final stage conditions (mainnet liberated) extrinsics should be activated
- [x] integration tests should not depend on sudo pallet (already solved by #4401) according to #3792
- [x] Ephesus Gateway WG should be discarded in favor of the APP WG
- [x] `update_curator_group_permission` call enabled in ephesus but it should be disabled in mainnet as the latest

**Comment by dobertRowneySr** [comment]
Conflict list
```
Auto-merging .github/workflows/run-network-tests.yml
Auto-merging Cargo.lock
CONFLICT (content): Merge conflict in Cargo.lock
Auto-merging bin/node/Cargo.toml
CONFLICT (content): Merge conflict in bin/node/Cargo.toml
Auto-merging bin/utils/chain-spec-builder/Cargo.toml
CONFLICT (content): Merge conflict in bin/utils/chain-spec-builder/Cargo.toml
Auto-merging chain-metadata.json
CONFLICT (content): Merge conflict in chain-metadata.json
Auto-merging cli/README.md
CONFLICT (content): Merge conflict in cli/README.md
Auto-merging cli/package.json
Auto-merging cli/src/Api.ts
Auto-merging cli/src/commands/fee-profile/addVideoComment.ts
CONFLICT (content): Merge conflict in cli/src/commands/fee-profile/addVideoComment.ts
Auto-merging cli/src/graphql/generated/schema.ts
Auto-merging joystream-node.Dockerfile
Auto-merging package.json
Auto-merging query-node/codegen/package.json
CONFLICT (content): Merge conflict in query-node/codegen/package.json
Auto-merging query-node/codegen/yarn.lock
CONFLICT (content): Merge conflict in query-node/codegen/yarn.lock
Auto-merging query-node/manifest.yml
Auto-merging query-node/mappings/package.json
CONFLICT (content): Merge conflict in query-node/mappings/package.json
Auto-merging query-node/mappings/src/common.ts
CONFLICT (content): Merge conflict in query-node/mappings/src/common.ts
Auto-merging query-node/mappings/src/content/channel.ts
CONFLICT (content): Merge conflict in query-node/mappings/src/content/channel.ts
Auto-merging query-node/mappings/src/proposals.ts
CONFLICT (content): Merge conflict in query-node/mappings/src/proposals.ts
Auto-merging query-node/schemas/content.graphql
Auto-merging runtime-modules/proposals/codex/src/lib.rs
Auto-merging runtime-modules/proposals/codex/src/tests/mock.rs
Auto-merging runtime-modules/proposals/codex/src/tests/mod.rs
Auto-merging runtime/Cargo.toml
CONFLICT (content): Merge conflict in runtime/Cargo.toml
Auto-merging runtime/src/integration/proposals/proposal_encoder.rs
Auto-merging runtime/src/lib.rs
CONFLICT (content): Merge conflict in runtime/src/lib.rs
Auto-merging tests/network-tests/package.json
Auto-merging tests/network-tests/src/Api.ts
Auto-merging tests/network-tests/src/graphql/generated/queries.ts
Auto-merging tests/network-tests/src/graphql/generated/schema.ts
Auto-merging tests/network-tests/src/scenarios/full.ts
CONFLICT (content): Merge conflict in tests/network-tests/src/scenarios/full.ts
Auto-merging types/src/augment/augment-api-consts.ts
Auto-merging types/src/augment/augment-api-errors.ts
Auto-merging types/src/augment/augment-api-events.ts
Auto-merging types/src/augment/augment-api-query.ts
Auto-merging types/src/augment/augment-api-tx.ts
Auto-merging types/src/augment/lookup.ts
CONFLICT (content): Merge conflict in types/src/augment/lookup.ts
Auto-merging types/src/augment/types-lookup.ts
CONFLICT (content): Merge conflict in types/src/augment/types-lookup.ts
Auto-merging yarn.lock
CONFLICT (content): Merge conflict in yarn.lock
Automatic merge failed; fix conflicts and then commit the result.
```
